test_files/graph/file.h

```diff
@@ -0,0 +1,20 @@
+
+
+
+#include "subfile1.h"
+
+
+
+
+#include "subfile2.h"
+
+
+
+
+#include "subfile3.h"
+
+
+#include "subsubfile2.h"
```

Comment:
Is there a reason why these includes are spread out across the file like this?

Reply:
No.

Reply:
Fixed.